src/Microsoft.ML.Onnx/SaveOnnxCommand.cs

```diff
@@ -134,7 +136,50 @@ private void GetPipe(OnnxContextImpl ctx, IChannel ch, IDataView end, out IDataV
                 transform = (source = transform.Source) as IDataTransform;
             }
 
-            Host.AssertValue(source);
+            Contracts.AssertValue(source);
+        }
+
+        internal static ModelProto ConvertTransformListToOnnxModel(OnnxContextImpl ctx, IDataView inputData, IDataView outputData,
+            LinkedList<ITransformCanSaveOnnx> transforms, HashSet<string> inputColumnNamesToDrop=null, HashSet<string> outputColumnNamesToDrop=null)
```

Comment:
>=null [](start = 142, length = 5)

Maybe ctrl-k-d to make VS format the file? What you think?

Reply:
Formatted ModelOperationsCatalog.cs, OnnxConversionTest.cs, and ProtabilityConversion.cs.

---
In reply to: [245428880](https://github.com/dotnet/machinelearning/pull/2013#discussion_r245428880) [](ancestors = 245428880)